src/clients/jobManagerWrapper.ts

```diff
@@ -46,35 +48,40 @@ export class JobManagerWrapper extends JobManagerClient {
     this.jobDomain = config.get<string>('externalClientsConfig.clientsUrls.jobManager.jobDomain');
   }
 
-  /**
-   * @deprecated The method should not be used
-   */
-  public async create(data: IWorkerInput): Promise<ICreateJobResponse> {
+  @withSpanAsyncV4
```

Comment:
i think the instrumentation gives us external requests span out of the box, please check if this decorator is really needed

Reply:
can you please move the trace to https://github.com/MapColonies/exporter-trigger/blob/728d70a7a8250d938d5372a69691f21d26767472/src/tasks/models/tasksManager.ts#L90